File: scratch/remove_dialog.py
import re
with open('c:/ERP/src/components/trims-store/trims-grn-form.tsx', 'r', encoding='utf-8') as f:
    content = f.read()
content = re.sub(r'\{\/\* Load PO Items Dialog \*\/}.*?<\/Dialog>', '', content, flags=re.DOTALL)
with open('c:/ERP/src/components/trims-store/trims-grn-form.tsx', 'w', encoding='utf-8') as f:
    f.write(content)

File: scratch/remove_dialog.js
const fs = require('fs');
let content = fs.readFileSync('c:/ERP/src/components/trims-store/trims-grn-form.tsx', 'utf-8');
content = content.replace(/\{\/\* Load PO Items Dialog \*\/}[\s\S]*?<\/Dialog>/g, '');
fs.writeFileSync('c:/ERP/src/components/trims-store/trims-grn-form.tsx', content, 'utf-8');
